<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task</class>
 <widget class="QDialog" name="task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>99</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Descrição :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="descriptionText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>60</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="typeLabel">
      <property name="text">
       <string>Tipo :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="typeLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Data :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
